server/protocols/webadmin-cli/src/main/java/org/apache/james/httpclient/DomainClient.java

```diff
@@ -36,4 +36,7 @@
     @RequestLine("DELETE /{domainToBeDeleted}")
     Response deleteADomain(@Param("domainToBeDeleted") String domainName);
 
+    @RequestLine("GET /{domainName}")
+    Response isExist(@Param("domainName") String domainName);
```

Comment:
doesExist

Reply:
ok, I think i should read more grammar about does Exist/ be Exist :3